red-recipes/qcat/1.1.0-3-gbde56b5/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set version = "v1.1.0_3_gbde56b5" %}
```

```suggestion
{% set version = "1.1.0_3_gbde56b5" %}
```
drop `v` to match versions in our other recipes.

---

I think that's fair. I've already had to mutate it from the original git version to eliminate the hyphens.